doc/man3/SSL_CTX_set_options.pod

```diff
@@ -275,6 +275,15 @@ RFC7627 Extended Master Secret option on TLS and DTLS connection.
 If this option is set, Extended Master Secret is disabled. Clients will
 not propose, and servers will not accept the extension.
 
+=item SSL_OP_NO_DTLS_RECORD_REPLAY_CHECK
+
+Turn off DTLS replay detect as described in RFC 6374 section 4.1.2.6.
+By default, replay detect is built in DTLS record handling. Some 
```

Comment:
make this "built into"

Reply:
> make this "built into"

@mattcaswell  is this the outstanding "nit pick". As I said I am unfamiliar how the process of a PR works. But couldnt we fix this on our own?

Reply:
right, this is a nitpick which is easily fixed. The one that is harder to do is my comment in the test/bad_dtls_test.c file - the test needs to be moved to a different file. This requires a bit more effort.

Reply:
Okay got it thx